tests/test_notifications.py

```diff
@@ -0,0 +1,841 @@
+from modularodm import Q
```

Comment:
NP: Imports should be grouped into system, third-party, and local source tree each of which are separated by a single empty line.